BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -107,6 +105,8 @@ final class LocationView: UIView,
         setupLayout()
         setupGradientLayer()
         addLongPressGestureRecognizer()
+        // Add a tap gesture recognizer to make the entire view tapable.
+        addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(becomeFirstResponder)))
```

Comment:
Shall we maybe only add this for version1 of the layout?

Reply:
Tested on both and it does not harm the `default` one, but I can change it to have it only for `version1`.